src/app/components/error/index.html

```diff
@@ -1,13 +1,196 @@
+            </sky-docs-visual-example-image>
+
+        </sky-docs-visual-example>
+        
+        <sky-docs-visual-example>
+        <sky-docs-visual-example-text>
+            <p>Don’t use an error when the status of specific content needs user attention, but the system has not necessarily broken. To notify the user of a broken status on a workflow or record, use an <a routerLink="/components/alert">alert</a> instead.</p>
+        </sky-docs-visual-example-text>
+          <sky-docs-visual-example-image>
+            <div class="sky-margin-stacked-content-section">
+              <img src="~/assets/img/guidelines/error/error-usage-3.png" />
+                <div class="caption caption-danger sky-text-danger">Don't use the error component to represent issuse with content that user would expect to address.</div>
+            </div>
+              <div>
+              <img src="~/assets/img/guidelines/error/error-usage-4.png" />
+                <div class="caption caption-success sky-text-success">Do use alert to call attention to issues that are intended to prompt user interaction.</div>
+            </div>
+          </sky-docs-visual-example-image>
+                
+        </sky-docs-visual-example>
+        
+        <sky-docs-visual-example>
+        <sky-docs-visual-example-text>
+          <p>Don’t use the security error type when a user doesn’t have permission to view a portion of the page such as a tile on a record page. Hide the tile from view instead.</p>
+        </sky-docs-visual-example-text>
+          <sky-docs-visual-example-image>
+            <div>
+              <img src="~/assets/img/guidelines/error/error-usage-5.png" />
+                <div class="caption caption-danger sky-text-danger">Don't use a security error in a portion of a page or modal. Hide the restricted content instead.</div>
+            </div>
+          </sky-docs-visual-example-image>
+                
+        </sky-docs-visual-example>
+        
+    </sky-demo-page-content>
+        
+    <sky-demo-page-content sectionHeading="Anatomy">
+        
+        <sky-docs-anatomy imagePath="~/assets/img/guidelines/error/error-anatomy.png">
+          <sky-docs-anatomy-item anatomyNumber="1">Title</sky-docs-anatomy-item>
+            <sky-docs-anatomy-item anatomyNumber="2" isOptional="true">Image</sky-docs-anatomy-item>
+            <sky-docs-anatomy-item anatomyNumber="3" isOptional="true">Description</sky-docs-anatomy-item>
+            <sky-docs-anatomy-item anatomyNumber="4" isOptional="true">Action</sky-docs-anatomy-item>
+          </sky-docs-anatomy>
+    
+    </sky-demo-page-content>
+        
+    <sky-demo-page-content sectionHeading="Options">
+        
+        <h3>Error type</h3>
+        
+        <sky-docs-visual-example>
+            <sky-docs-visual-example-text>
+              <p>Use the broken error type in situations where the underlying content doesn’t load or when there is an issue reaching the server, such as 500 errors.</p>
```

Comment:
I'd tweak the end of the sentence to: "... or when an issue such as a 500 error reaches the server."

Also, should `error` be a code snippet since it's referring to the type that is specified with the `errorType` property? I think we're probably fine not using a code snippet since this is a generic reference, but I figured it was worth asking. Same question for the other types.

Reply:
I think we should avoid code snippets in the "Design" tab, to avoid discussing specific implementation when we should be discussing behavior.
BUT, would it be worthwhile to put the type name itself in quotations?

> Use the "broken" error type in situations where...

Reply:
We could discuss that pattern of quote marks, but my initial instinct is to not use them in this scenario. I think it would add ambiguity, but it's worth talking to Todd and others if you think it would make things more clear. Personally, I'd lean toward generic references like you have here without the quote marks. When I noticed that the types corresponded to the properties on the Development tab, I just wanted to make sure we were intentionally not using code snippets.